sbt/src/sbt-test/tests/fork-options-custom-config/test

```diff
@@ -0,0 +1,3 @@
+> check
+> test
+-> Slowtests / test
```

Comment:
This doesn't fail. Either forking doesn't work properly _or_ the `javaOptions` come from some place far far away.

I used to add  these before the `check` command:

```
> show SlowTests / forkOptions
> show SlowTests / run / forkOptions
> show SlowTests / test / forkOptions
> inspect SlowTests / forkOptions
> inspect SlowTests / run / forkOptions
> inspect SlowTests / test / forkOptions
> show test / forkOptions
```